app/src/main/res/values/strings.xml

```diff
@@ -777,4 +762,5 @@
     <string name="translate_descriptions_maybe_later">Maybe later</string>
     <string name="title_descriptions_onboarding_got_it">Got it</string>
     <string name="translate_descriptions_onboarding_got_it">Got it</string>
+    <string name="android_app_edit_help_url">https://www.mediawiki.org/wiki/Wikimedia_Apps/Suggested_edits</string>
```

Comment:
As I mentioned in Slack, let's make this point to the mobile version of the page. (`m.` instead of `www.`)

Reply:
I left it like that because it was automatically opening the mobile-compatible page with m. . However, just in case it is only chrome that does that, I have changed to m. .